lib/main.dart

```diff
@@ -81,7 +83,11 @@ class _AppState extends State<App> {
             supportedLocales: const [
               Locale('en', ''), // English, no country code
               Locale('es', ''), // Spanish, no country code
-              Locale('zh', ''), // Chinese (Mandarin), no country code
+              Locale.fromSubtags(languageCode: 'zh'), // generic Chinese 'zh'
+              Locale.fromSubtags(
+                languageCode: 'zh',
+                countryCode: 'HK',
+              ), // generic traditional Chinese 'zh_Hant'
```

Comment:
```suggestion
              Locale.fromSubtags(languageCode: 'zh'), // Simplified Chinese, no country code
              Locale.fromSubtags(
                languageCode: 'zh',
                countryCode: 'HK',
              ), // Chinese, Hong Kong
```

Reply:
BTW is that equivalent to 
```suggestion
              Locale('zh', ''), // Chinese (Mandarin), no country code
              Locale('zh', 'HK'), // Chinese (Cantonese), Hong Kong
```